configurations/local.template.ini

```diff
@@ -137,8 +137,8 @@ metadata_sphinx_num_of_int_fields = 10
 ;the range is used when checking service actions permissions and allowing to access certain 
 ;services without KS from the internal servers
 
-partner_otp_internal_ips = @IP_RANGE@
-otp_required_partners[] = -2
+;partner_otp_internal_ips = @IP_RANGE@
```

Comment:
We have the settings.remoteAddrHeaderSalt in place. However, since we do not set up a 2 factor auth by default, there is no reason at all why these directives should be enabled..
The internal_ip_range remains enabled of course but partner_otp_internal_ips relates ONLY to the OTP mechanism, which again, is not activated by default.